### DISCUSSION TITLE
My account got suspended 3 weeks ago

### DISCUSSION THREAD

**Discussion by abdulrehmanwaseem** [community/community#140015] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi GitHub Community,

My account got suspended 3 weeks ago. I created a new account (this) to access this page and start a discussion. The username of my account that got suspended is abdulrehman-code.

I submitted a ticket (#2985302) to request an appeal and reinstatement on 5 September. Today is 28 September. Unfortunately, there has been no response to the ticket yet.

I really need access to my account for working on professional and personal projects. I am also in the middle of creating a startup and I had an organization in that account.

I understand the importance of adhering to GitHub’s Terms of Service and Community Guidelines. If there were any inadvertent violations on my part, I sincerely apologize and am committed to ensuring full compliance in the future.

Please review my case and consider reinstating my account. If further information is needed, I am more than willing to provide it.

Thank you for your understanding and assistance.

Sincerely,
Abdul Rehman Waseem

GitHub Username: abdulrehman-code
Email: abdulrehmanwaseem2024@gmail.com

**Comment by ebndev** [comment, marked as answer]
Hi @abdul-rehman22,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
